js/actualtime.js

```diff
@@ -1,257 +1,265 @@
+               var _at = 'left+20 bottom-20';
+               //calculate relative dialog position
+               $('.message_result').each(function () {
+                  var _this = $(this);
+                  if (_this.attr('aria-describedby') != 'message_result') {
+                     _of = _this;
+                     _at = 'right top-' + (10 + _this.outerHeight());
+                  }
+               });
+               $('#message_result').dialog({
+                  dialogClass: 'message_after_redirect ' + result['class'],
+                  minHeight: 40,
+                  minWidth: 200,
+                  position: {
+                     my: 'left bottom',
+                     at: _at,
+                     of: _of,
+                     collision: 'none'
+                  },
+                  autoOpen: false,
+                  show: {
+                     effect: 'slide',
+                     direction: 'down',
+                     'duration': 800
+                  }
+               })
+                  .dialog('open');
+               $(document.body).on('click', function (e) {
+                  if ($('#message_result').dialog('isOpen')
+                     && !$(e.target).is('.ui-dialog, a')
+                     && !$(e.target).closest('.ui-dialog').length) {
+                     $('#message_result').dialog('close');
+                     // redo focus on initial element
+                     e.target.focus();
+                  }
+               });
+            });
          }
+      });
+   }
+
+   this.init = function(ajax_url) {
+      window.actualTime.ajax_url = ajax_url;
+      if (!$("#message_result").length) {
+         $("body").append("<div id='message_result'></div>");
       }
-   });
 
-});
+      // Initialize
+      jQuery.ajax({
+         type: 'GET',
+         url: window.actualTime.ajax_url + '?footer',
+         dataType: 'json',
+         success: function (result) {
+            symb_d = result['symb_d'];
+            symb_day = result['symb_day'];
+            symb_days = result['symb_days'];
+            symb_h = result['symb_h'];
+            symb_hour = result['symb_hour'];
+            symb_hours = result['symb_hours'];
+            symb_min = result['symb_min'];
+            symb_minute = result['symb_minute'];
+            symb_minutes = result['symb_minutes'];
+            symb_s = result['symb_s'];
+            symb_second = result['symb_second'];
+            symb_seconds = result['symb_seconds'];
+            text_warning = result['text_warning'];
+            text_pause = result['text_pause'];
+            text_restart = result['text_restart'];
+            text_done = result['text_done'];
+            popup_div = result['popup_div'];
+
+            if (result['ticket_id']) {
+               window.actualTime.startCount(result['task_id'], result['time']);
+               window.actualTime.showTimerPopup(result['ticket_id']);
+            }
+         }
+      });
+   }
+}();
```

Comment:
Hi @cconard96 thanks for your contribution.
I found that there is a global variable for javascript with the plugin paths `GLPI_PLUGINS_PATH`.
Adding this code at the end of the file will show the timer on all pages and should not affect the documents
`$(document).ready(function(){
var url = CFG_GLPI.root_doc+"/"+GLPI_PLUGINS_PATH.actualtime+"/ajax/timer.php";
window.actualTime.init(url);
});`

Reply:
Good catch. I guess I have some changes to make with my own plugins now too.